src/index.js

```diff
@@ -1,8 +1,11 @@
 import { applyMiddleware } from 'redux'
-import isPlainObject from 'lodash.isplainobject'
 
 const isFunction = arg => typeof arg === 'function'
 
+const isPlainObject = function (obj) {
+  return Object.prototype.toString.call(obj) === '[object Object]'
```

Comment:
This will only check if supplied value is object or not. For plain object you also need to check prototype.
`Object.getPrototypeOf(value) === Object.prototype`

For Ex: 
var objA = {name: 'test'}; // This is plain object
var objB = Object.create(objA); // This is not

Reply:
I copied the code from loads itself so it should do the same as before.